replica/database.hh

```diff
@@ -1978,3 +1978,25 @@ public:
         return semaphore().obtain_permit(schema, description, cf.estimate_read_memory_cost(), timeout, std::move(trace_ptr));
     }
 };
+
+namespace replica {
+
+class database_schema_describe_helper : public schema_describe_helper {
+    database& _db;
```

Comment:
Since all three methods can be `const`, this reference can be `const` as well

Reply:
v2 uses `data_dictionary::database` instead.